person.rb

```diff
@@ -19,6 +23,10 @@ def of_age?
   private :is_of_age?
```

Comment:
- [ optional ] Please as mentioned by the previous reviewer kindly change the `:is_of_age?` to `of_age?`

Reply:
Thank you very much, I keep on missing that 😆